src/theme/index.ts

```diff
@@ -10,11 +10,17 @@ export const THEME = {
     white: '#ffffff',
     black: '#000000',
     font: '#ffffff',
+    fontDark: '#000000',
```

Comment:
I'm not sure if this PR is the right place to suggest this, but it seems like there are 3 separate naming schemes for colors in the theme:
- By typical color names - white, black
- By conceptual color name - brand (it's not a color name, but it means a specific color)
- By use - font, menuItem, etc. (where they are used, no info on the actual color)

I would recommend picking one and sticking with it.

My personal recommendation would be to name by color like blue, darkBlue or even use a numbering scheme like [Tailwind CSS](https://tailwindcss.com/docs/customizing-colors) does - blue-100 through blue-900, light to dark. Comments above the colors could clarify their intended use. That would make it clear what the color is when you see it in context in the code.

But picking one scheme would be better, regardless of which one it is.

Reply:
I agree, :D I think it is best if we discuss this separately indeed. I added this as a feature request on discord.

Here are a few comments about the matter:
Although the tailwind color scheme is good, I dislike it because it is almost the same as just using the colors inline.
It doesn't define true colors for specific things.

My reasoning is always the following:
- Fonts by use is the way to go, this makes sure when you use the color you know what it is for.
- A disadvantage of this is, in large applications you tend to have 100+ different use color names. Which is why conceptional color names are used for the more common brand specific colors
- Typical colors contribute to the same point above, although they are not that important and could be removed to make it more clear.

As said before, moving this to discord